ui/src/reducers/app.js

```diff
@@ -33,35 +33,40 @@ export const cloneAppData = (appData) => {
 
 export default (state = initialState, action) => {
   switch (action.type) {
-    case APP_VERSIONS_SORT:
+    case APP_VERSIONS_SORT: {
```

Comment:
Good catch & fix in adding a block scope for each switch case implementation :+1:

Reply:
This came from @craicoverflow . Thumbs up to him